pubspec.yaml

```diff
@@ -5,14 +5,14 @@ version: 3.2.2
 homepage: https://github.com/teranetsrl/oauth2_client
 repository: https://github.com/teranetsrl/oauth2_client
 environment:
-  sdk: '>=2.14.0 <3.11.0'
+  sdk: '>=2.14.0 <4.0.0'
 
 dependencies:
   flutter:
     sdk: flutter
   crypto: ^3.0.3
-  flutter_secure_storage: ^8.0.0
-  flutter_web_auth_2: ^2.2.1
+  flutter_secure_storage: ^9.0.0
+  flutter_web_auth_2: ^3.0.1
```

Comment:
Can you [put the latest version](https://github.com/ThexXTURBOXx/flutter_web_auth_2/releases/tag/3.0.3)?

Reply:
No, because the caret already resolves that and hence it is unnecessary to constrain the version range more than that